dashboard/app/views/levels/_blockly.html.haml

```diff
@@ -1,6 +1,7 @@
 -# TODO(JL): conditionally load on client
 - if browser.chrome? && browser.version.to_s.to_i == 34
-  %script{src: '/shared/js/chrome34Fix.js'}
+  :javascript
+    window.dashboard.isChrome34 = true;
```

Comment:
Yeah, if there are things like this that we want to separate out, we can use factor-bundle or partition-bundle to generate multiple bundles, and still get the other benefits of browserify/linting/etc.